File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        int p = 0, y = 0;
        String ss = s.toLowerCase();

        for (int i = 0; i < ss.length(); i++) {
            if (ss.charAt(i) == 'p') {
                p++;
            } else if (ss.charAt(i) == 'y') {
                y++;
            }
        }
        return p == y;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    p_count = 0
    y_count = 0
    for i in list(s):
        if i.lower() == 'p':
            p_count += 1
        elif i.lower() == 'y':
            y_count += 1
    return p_count == y_count